src/hisp/plamsa_data_handling/main.py

```diff
@@ -223,15 +223,15 @@ def get_heat(self, pulse: Pulse, bin: SubBin | DivBin, t_rel: float) -> float:
             raise ValueError(f"Invalid pulse type {pulse.pulse_type}")
 
         if pulse.pulse_type == "FP":
-            heat_total = data["heat_total"][bin_index]
+            heat_total = data["heat_total"][bin_index]+0.11e+06 # photon radiation heat adjustment
             heat_ion = data["heat_ion"][bin_index]
             if isinstance(bin, SubBin):
                 heat_val = heat_total - heat_ion * (1 - bin.wetted_frac)
             else:
                 heat_val = heat_total
         elif pulse.pulse_type == "RISP":
             if isinstance(bin, SubBin):
-                heat_total = data["heat_total"]
+                heat_total = data["heat_total"]+0.11e+06 # photon radiation heat adjustment
```

Comment:
🪄 Magic number alert

```suggestion
                photon_radiation_heat = 0.11e6  # W/m2
                heat_total = data["heat_total"]+photon_radiation_heat 
```